packages/react-composites/tests/browser/call/hermetic/RoomsCallScreen.test.ts

```diff
@@ -73,87 +73,60 @@ test.describe('Rooms CallScreen tests for different roles', async () => {
 });
 
 test.describe('Rooms Participant RemoveButton tests for different roles', async () => {
-  test('Remove button is enabled for Presenter', async ({ page, serverUrl }, testInfo) => {
+  test.only('Remove button is enabled for Presenter', async ({ page, serverUrl }, testInfo) => {
```

Comment:
Remove `test.only` (that's probably why CI failed).

Reply:
done